### ISSUE TITLE
Ember Global deprecation warning due to old version of ember-cli-babel

### ISSUE THREAD

**Issue by rahulk94** [mitchlloyd/ember-screen#9]
We're looking to upgrade our application from Ember 3.24 to 3.28 (latest LTS) and have come across the following deprecation warnings due to `ember-screen`'s use of `ember-cli-babel@6`.
```
WARNING: [DEPRECATION] [DEPRECATION] Usage of the Ember Global is deprecated. You should import the Ember module or the specific API instead.

See https://deprecations.emberjs.com/v3.x/#toc_ember-global for details.

Usages of the Ember Global may be caused by an outdated ember-cli-babel dependency. The following steps may help:

* Upgrade the following addons to the latest version:
  * ember-screen

### Important ###

In order to avoid repeatedly showing the same deprecation messages, no further deprecation messages will be shown for usages of the Ember Global until ember-cli-babel is upgraded to v7.26.6 or above.

To see all instances of this deprecation message, set the `EMBER_GLOBAL_DEPRECATIONS` environment variable to "all", e.g. `EMBER_GLOBAL_DEPRECATIONS=all ember test`.

### Details ###

Prior to v7.26.6, ember-cli-babel sometimes transpiled imports into the equivalent Ember Global API, potentially triggering this deprecation message indirectly, even when you did not observe these deprecated usages in your code.

The following outdated versions are found in your project:

* ember-cli-babel@6.18.0, currently used by:
  * ember-maybe-import-regenerator@0.1.6 (Dormant)
    * Depends on ember-cli-babel@^6.0.0-beta.4
  * ember-screen@1.0.0
    * Depends on ember-cli-babel@^6.6.0

Note: Addons marked as "Dormant" does not appear to have any JavaScript files. Therefore, even if they are using an old version ember-cli-babel, they are unlikely to be the culprit of this deprecation and can likely be ignored.

```
(above logs from a freshly generated Ember 3.28 app)

Is this package still maintained? and if so, is this something that could be upgraded?

**Comment by mitchlloyd** [comment]
Yep this seems fixable to me. I’m just starting a 4 day road trip and won’t
be able to get to a computer until 11/23 or 11/24. If you can put up a PR
to fix this I could take a look then.

**Comment by rahulk94** [comment]
Cool I've raised https://github.com/mitchlloyd/ember-screen/pull/10 to fix this deprecation.

**Comment by mitchlloyd** [comment]
Closed with #10, but without a release yet. I'm planning to bundle any embroider work in the next release in case there is a desirable approach with a breaking change.
